POSTGAP.py

```diff
@@ -317,6 +317,7 @@ def get_options():
     parser.add_argument('--hdf5',help='Location of eQTL HDF5 file')
     parser.add_argument('--sqlite',help='Location of eQTL sqlite file')
     parser.add_argument('--output2', help='gene-cluster association output file')
+    parser.add_argument('--eqtl_response_size', help='Number of items returned in one call to EQTL, default is 200, max is 1000')
```

Comment:
What is the type of this argument? String or int? This is ambiguous in this commit.

Reply:
Also, you could add a default value on this line, this will simplify things.